packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,376 @@ components:
+        tracks:
+          type: array
+          description:
+            List of tracks associated with the asset when the format contemplate
+            them (e.g. mp4)
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              kind:
+                type: string
+                description: Kind of track
+                example: video
+              width:
+                type: number
+                description: Width of the track - only for video tracks
+                example: 1920
+              height:
+                type: number
+                description: Height of the track - only for video tracks
+                example: 1080
+              fps:
+                type: number
+                description: Frame rate of the track - only for video tracks
+                example: 30
+              id:
+                type: string
+                description: ID of the track
+                example: ABC123
+              duration:
+                type: number
+                description: Duration of the track in seconds
+                example: 23.8238
+              bitrate:
+                type: number
+                description: Bitrate of the track in bits per second
+                example: 1000000
+              channels:
+                type: number
+                description: Amount of audio channels in the track
+                example: 2
+              sampleRate:
+                type: number
+                description:
+                  Sample rate of the track in samples per second - only for
+                  audio tracks
+                example: 44100
+              bitDepth:
+                type: number
+                description: Bit depth of the track - only for audio tracks
+                example: 16
+              codec:
+                type: string
+                description: Codec of the track
+                example: aac
+        profiles:
+          type: object
+          additionalProperties: false
+          description: HLS Profile manifests associated with the asset
+          properties:
+            kind:
+              type: string
+              description: Type of the profile
+              example: manifest
+            name:
+              type: string
+              description: Filename of the manifest
+              example: variant_1080.m3u8
+            width:
+              type: number
+              description: width of the profile
+              example: 1920
+            height:
+              type: number
+              description: height of the profile
+              example: 1080
+            fps:
+              type: number
+              description: Frame rate of the profile
+              example: 30
+            id:
+              type: string
+              description: ID of the profile
+              example: ABC123
+            duration:
+              type: number
```

Comment:
Instead of a single example, I think this should have/be an `enum` instead and we list all the possible values here